mods/hide-pin.wh.cpp

```diff
@@ -0,0 +1,125 @@
+// ==WindhawkMod==
+// @id              hide-pin
+// @name            Hide Pin (in Explorer Navigation Pane)
+// @description     Hides pin icon in Explorer navigation pane
+// @version         1.0
+// @author          @danalec
+// @github          https://github.com/danalec
+// @include         explorer.exe
+// ==/WindhawkMod==
+
+// ==WindhawkModReadme==
+/*
+Removes pin icons from File Explorer's navigation pane by intercepting the tree view's item state queries.
+*/
```

Comment:
How about adding a screenshot to show what exactly the mod does?

Reply:
You can use imgur.com or GitHub, see the documentation:
https://github.com/ramensoftware/windhawk/wiki/Creating-a-new-mod#readme